index.js

```diff
@@ -110,16 +107,13 @@ exports.main = function main() {
       .then(() => getProjectReadme(projectDirName))
       .then(projectReadme => fs.writeFile(path.join(projectPath, "README.md"),
                                           stripAnsi(projectReadme)))
-      .then(() => getProjectCreatedMessage(projectPath))
-      .then(console.log);
+      .then(() => getProjectCreatedMessage(projectPath));
   }, error => {
     if (error.code === "EEXIST") {
       const msg = `Unable to create a new WebExtension: ${chalk.bold.underline(projectPath)} dir already exist.`;
-      console.error(`${chalk.red(msg)}\n`);
-      process.exit(1);
+      throw new UsageError(msg);
     }
   }).catch((error) => {
-    console.error(error);
-    process.exit(1);
+    throw error;
```

Comment:
@saintsebastian we can omit this catch, it is basically doing what it is going to happen if we omit it.